### PR TITLE
Edit Dashlet views fix.

### PR DESCRIPTION
**Pull request by e-reeley**
## Description

The edit dashlet viewhad issues regarding overlapping fields and displays.

## How To Test This

Click on the edit button on a dashlet, the contents should be in line and visible.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Final checklist

- [x] My code follows the code style of this project found [here](https://suitecrm.com/wiki/index.php/Coding_Standards).
- [ ] My change requires a change to the documentation.
- [x] I have read the [**How to Contribute**](https://suitecrm.com/wiki/index.php/Contributing_to_SuiteCRM) guidelines.

### CODE REVIEW

**Comment by pgorod** [comment]
Does this fix Issue #4176 ?

**Comment by e-reeley** [comment]
Hi @pgorod , the issue fixed here was to fix a styling issue within the edit dashlet views.

Before this update, there seemes to be some overlapping fields, text boxes, drop down boxes. 
This aims to allign them.

I will take a look into the linked issue, thank you for pointing this out.

**Comment by e-reeley** [comment]
Hi @pgorod , I have a fix for issue #4176.

**Comment by Dillon-Brown** [comment]
Assessed

File: themes/SuiteP/css/suitep-base/dashboard.scss [hunk 1]
```diff
@@ -653,10 +655,10 @@ td.dashlet-title > h3 > span {
   }
   .yui-skin-sam.masked #dlg.SuiteP-configureDashlet.yui-module.yui-overlay.yui-panel .bd form table.edit tr td[valign=top] {
     float: left;
-    width: 25%;
+    width: 25%;//changed from 25%
```
Comment (PedroErnst) [inline themes/SuiteP/css/suitep-base/dashboard.scss]: Changed from 25% to 25% ?

**Comment by Dillon-Brown** [comment]
Assessed

**Pull request merged**